EIPS/eip-1261.md

```diff
@@ -364,7 +364,7 @@ Membership Verification Token ERC1261 -- a reference implementation
 1. ERC-20 Token Standard. https://github.com/ethereum/EIPs/blob/master/EIPS/eip-20.md
 1. ERC-165 Standard Interface Detection. https://github.com/ethereum/EIPs/blob/master/EIPS/eip-165.md
```

Comment:
And this too.

Reply:
Fixed at f87cc3a71b4e83627d0b83ce00ff7344cfa7ce35